server/app/main.py

```diff
@@ -1,8 +1,20 @@
+from typing import Any
+
+import uvicorn
 from fastapi import FastAPI
+from grpc_manager.base import GrpcManager
 
 app = FastAPI()
 
 
+if __name__ == "__main__":
+    grpc_manager = GrpcManager()
+    uvicorn.run("main:app", host="0.0.0.0")
```

Comment:
uvicorn's default port is 8000. Run bots in another port by passing another arg port = 9000 for ex